### DISCUSSION TITLE
スマホやタブレットから閲覧した場合でも目次を表示してほしい

### DISCUSSION THREAD

**Discussion by yarakos95** [increments/qiita-discussions#375] [要望・アイデア・不具合]
スマホやタブレットから記事を閲覧した場合、目次が表示されません。また、PC であっても幅を狭くすると目次を見ることは出来なくなってしまいます。目次が表示されないと、記事の全体像が不明瞭で少し読みづらく感じます。

スマホ (Chrome) で閲覧した様子：

> https://qiita.com/Qiita/items/c686397e4a0f4f11683d

|  ページ上部  | ページの途中 |
| :----------: | :----------: |
| ![Qiita (1)] | ![Qiita (2)] |

同系のサイトとして Zenn を見てみると、上部のメニューバーに目次があり、ページの途中からでも目次を確認することが出来ます。Qiita も同じように下部にあるメニューバー内から目次を表示させることが出来ると良いと思います。

> https://zenn.dev/zenn/articles/markdown-guide

[Qiita (1)]: https://user-images.githubusercontent.com/75878492/207052191-bf1b21c2-f072-4d83-bfe8-02addf624eac.jpg
[Qiita (2)]: https://user-images.githubusercontent.com/75878492/207052192-036388ba-1a29-4842-ab37-e798abe68b43.jpg
[Zenn (1)]: https://user-images.githubusercontent.com/75878492/207052176-6667ab66-7a84-4c76-bcf5-bf09d87bb45c.jpg
[Zenn (2)]: https://user-images.githubusercontent.com/75878492/207052186-5fd3e18f-093b-407d-aa30-38a3251df366.jpg
[Zenn (3)]: https://user-images.githubusercontent.com/75878492/207052189-4530f07e-ad50-411a-995b-8f1f70139315.jpg

**Comment by tanaka-asami415** [comment]
@Yarakashi-Kikohshi 
ご意見ありがとうございます！🙏
こちらは実装したいと思いますので、
現在進めているプロジェクトの進捗と照らし合わせた上で、対応したいと思います！💪✨
